packages/calcite-components/src/components/shell-panel/usage/With-custom-element.md

```diff
@@ -8,7 +8,7 @@ Add `calcite-match-height` to a wrapping element to ensure proper height, scroll
     <calcite-action text="Layers" icon="layers"></calcite-action>
   </calcite-action-bar>
   <your-custom-element class="calcite-match-height">
-    <calcite-panel> ... </calcite-panel>
+    <calcite-panel> <img src="https://placebear.com/g/600/600" alt="placeholder" /> </calcite-panel>
```

Comment:
Great catch, I will change to be blank so it gets skipped over because I'm not sure if the placebear images will always stay the same. I think they do but on the off chance they don't I don't want to have a description that doesn't have anything to do with the image.